perfkitbenchmarker/package_managers.py

```diff
@@ -115,11 +115,39 @@ def GetServiceName(self, package_name):
     """
     pass
 
+  def SetupProxy(self):
+    """
+        This function setup proxy if your cloud is currently installed behind proxy server
+    """
+    env_file = "/etc/environment"
+
+    commands = []
+
+    add_proxy_conf = "sudo touch %s" % env_file
+    commands.append(add_proxy_conf)
+
+    if FLAGS.http_proxy:
+        add_http_proxy = "echo -e 'http_proxy=%s' | sudo tee -a %s" % (FLAGS.http_proxy, env_file)
+        commands.append(add_http_proxy)
+
+
+    if FLAGS.https_proxy:
+        add_https_proxy = "echo -e 'https_proxy=%s' | sudo tee -a %s" % (FLAGS.https_proxy, env_file)
+        commands.append(add_https_proxy)
+
+    if FLAGS.ftp_proxy:
+        add_ftp_proxy = "echo -e 'ftp_proxy=%s' | sudo tee -a %s" % (FLAGS.ftp_proxy, env_file)
+        commands.append(add_ftp_proxy)
+
+    self.RemoteCommand(";".join(commands))
```

Comment:
Add a `if commands:` here, should only execute if there's something to do. (Assuming the `touch` is removed.)